Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04 as builder
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+RUN echo 'tzdata tzdata/Areas select Etc' | debconf-set-selections; \
+    echo 'tzdata tzdata/Zones/Etc select UTC' | debconf-set-selections; \
```

Comment:
What's this for? I feel like this could use an explanation (in the Dockerfile).

Reply:
It has now been explained 👍🏻